src/DeviceProvisioningServices/DeviceProvisioningServices.sln

```diff
@@ -11,8 +11,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DeviceProvisioningServices.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Accounts", "..\Accounts\Accounts\Accounts.csproj", "{142D7B0B-388A-4CEB-A228-7F6D423C5C2E}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IotHub", "..\IotHub\IotHub\IotHub.csproj", "{78770A60-B18A-4442-A982-0CEE0356F8DB}"
```

Comment:
@anusapan what's the reason for removing this reference? The DPS tests use the `Az.IotHub` module, so we want to make sure that the module is built whenever the DPS solution is built

Reply:
changes reverted